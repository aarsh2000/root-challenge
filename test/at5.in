Driver Snail
Trip Snail 23:30 24:00 2.5