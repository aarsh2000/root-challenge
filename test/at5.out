Snail: 3 miles @ 5 mph
